Patches/turk's guns/Patch_Guns.xml

```diff
@@ -0,0 +1,317 @@
+			<burstShotCount>6</burstShotCount>
+			<ticksBetweenBurstShots>6</ticksBetweenBurstShots>
+			<warmupTime>1.0</warmupTime>
+			<range>55</range>
+			<soundCast>Shot_AssaultRifle</soundCast>
+			<soundCastTail>GunTail_Medium</soundCastTail>
+			<muzzleFlashScale>9</muzzleFlashScale>
+		  </Properties>
+		  <AmmoUser>
+		    <magazineSize>90</magazineSize>
+			<reloadTime>4.9</reloadTime>
+			<ammoSet>AmmoSet_556x45mmNATO</ammoSet>
+		  </AmmoUser>
+		  <FireModes>
+		    <aiAimMode>SuppressFire</aiAimMode>
+            <aiUseBurstMode>true</aiUseBurstMode>
+            <aimedBurstShotCount>5</aimedBurstShotCount>
+		  </FireModes>
+		  <weaponTags>
+		    <li>IndustrialGunAdvanced</li>
+		  </weaponTags>
+		  <researchPrerequisite>PrecisionRifling</researchPrerequisite>
+		</li>
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+		  <defName>Gun_Skorpion</defName>
+		  <statBases>
+		    <WorkToMake>32000</WorkToMake>
+		    <SightsEfficiency>1.0</SightsEfficiency>
+			<ShotSpread>0.17</ShotSpread>
+			<SwayFactor>0.65</SwayFactor>
+			<Bulk>2.70</Bulk>
+			<Mass>1.30</Mass>
+			<RangedWeapon_Cooldown>0.37</RangedWeapon_Cooldown>
+		  </statBases>
+		  <costList>
+		  <Steel>30</Steel>
+		  <ComponentIndustrial>4</ComponentIndustrial>
+		  </costList>
+		  <Properties>
+		    <recoilAmount>1.86</recoilAmount>
+		    <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+			<hasStandardCommand>true</hasStandardCommand>
+			<defaultProjectile>Bullet_32ACP_FMJ</defaultProjectile>
+			<burstShotCount>6</burstShotCount>
+			<ticksBetweenBurstShots>4</ticksBetweenBurstShots>
+			<warmupTime>0.6</warmupTime>
+			<range>25</range>
+			<soundCast>Shot_MachinePistol</soundCast>
+			<soundCastTail>GunTail_Light</soundCastTail>
+			<muzzleFlashScale>9</muzzleFlashScale>
+		  </Properties>
+		  <AmmoUser>
+		    <magazineSize>20</magazineSize>
+			<reloadTime>4</reloadTime>
+			<ammoSet>AmmoSet_32ACP</ammoSet>
+		  </AmmoUser>
+		  <FireModes>
+		    <aiAimMode>Snapshot</aiAimMode>
+            <aiUseBurstMode>false</aiUseBurstMode>
+            <aimedBurstShotCount>3</aimedBurstShotCount>
+		  </FireModes>
+		  <researchPrerequisite>BlowbackOperation</researchPrerequisite>
+		</li>
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+		  <defName>Gun_MX21Four</defName>
+		  <statBases>
+		    <WorkToMake>60000</WorkToMake>
+		    <SightsEfficiency>1.0</SightsEfficiency>
+			<ShotSpread>0.06</ShotSpread>
+			<SwayFactor>2.05</SwayFactor>
+			<Bulk>9.32</Bulk>
+			<Mass>13.20</Mass>
+			<RangedWeapon_Cooldown>0.45</RangedWeapon_Cooldown>
+		  </statBases>
+		  <costList>
+		  <Steel>150</Steel>
+		  <ComponentIndustrial>12</ComponentIndustrial>
+		  </costList>
+		  <Properties>
+		    <recoilAmount>0.77</recoilAmount>
+		    <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+			<hasStandardCommand>true</hasStandardCommand>
+			<defaultProjectile>Bullet_556x45mmNATO_FMJ</defaultProjectile>
```

Comment:
I think the heavy minigun should be 7.62 NATO, modelled after the actual minigun. For this one, I think you can probably copy-paste the stats for the vanilla minigun if you like.

Reply:
sounds good